### ISSUE TITLE
Safari+VoiceOver: autosuggest doesn't announce aria-activedescendant changes

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1629]
As described in [this blog post](https://nolanlawson.com/2019/11/05/what-ive-learned-about-accessibility-in-spas/), the autosuggestion list in the compose box currently doesn't work quite right in Safari with VoiceOver. It does work fine in NVDA with Firefox though.

Background: https://github.com/nolanlawson/pinafore/pull/1513#issue-320087960

I think it might be related to this WebKit bug: https://bugs.webkit.org/show_bug.cgi?id=167671
